src/www/app/app.ts

```diff
@@ -468,14 +462,26 @@ export class App {
   private showServerRenamed(event: events.ServerForgotten) {
     const server = event.server;
     console.debug('Server renamed');
-    this.serverListEl.getServerCard(server.id).serverName = server.name;
+    this.updateUiServer(server.id, {name: server.name});
     this.rootEl.showToast(this.localize('server-rename-complete'));
   }
 
   // Helpers:
 
+  private makeUiServer(server: Server): ServerCardModel {
```

Comment:
personally I'd do something like `serverCardModelFromServer`. TS helps here, but at first glance I'd think we were creating html elements